media/js/firefox/desktop/intro-anim.js

```diff
@@ -5,9 +5,11 @@
 (function(Mozilla, $) {
     'use strict';
 
+    var client = Mozilla.Client;
+
     function cutsTheMustard() {
         // Bug (1083079) fixed but effects Firefox 35
-        if (window.isFirefox() && window.getFirefoxMasterVersion() === 35 && $('html').hasClass('osx')) {
+        if (client.isFirefox && client.FirefoxMajorVersion === 35 && client.platform === 'osx') {
```

Comment:
Firefox 35 is quite old now, I think we're good to remove this check as the bug has been fixed for a while :+1: